pipelines/_ingredients/ApVerify.yaml

```diff
@@ -3,6 +3,8 @@
 description: Fully instrumented AP pipeline.
 imports:
   - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
+    exclude:
+      - qaMetrics
```

Comment:
This is not correct; the `ApVerify` pipeline should be the `ApPipe` pipeline _plus_ metrics, without the need for any further tweaks. Excluding tasks here on the grounds that they'd be included in the instrument specialization will create a great deal of confusion and maintenance difficulty, especially since they won't appear in graphs of the pipeline.

If you're getting duplicate imports, that suggests there's something broken with the pipeline definition, and it's better to fix it then patch it.